src/components/Card/Card.jsx

```diff
@@ -8,9 +8,10 @@ const Card = (props) => {
       <h2>{title}</h2>
       <img src={image} alt={title} />
 
-      <a href={url}> {title}</a>
-
-
+      <a href={url} className="cardhref ">
+        {" "}
```

Comment:
This line isn't needed a plain old space should do the trick, or if you need more than one space, you can use &nbsp; which forces HTML to insert a space (normally if you have more than one space next to eachother, HTML only renders one